stories/Collapse/CollapseNestedExample.js

```diff
@@ -79,4 +79,8 @@ const CollapseNestedExample = () => {
   )
 }
 
-export default CollapseNestedExample
+const CollapseNestedExampleWithHook = props => (
+  <CollapseNestedExample {...props} />
+)
+
+export default CollapseNestedExampleWithHook
```

Comment:
This is not required. Can you revert it?
It breaks the `Source Code` block in the addon-info panel.